cmd/run/run.go

```diff
@@ -60,16 +61,256 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+const (
+	datastoreEngineFlag = "datastore-engine"
+	datastoreURIFlag    = "datastore-uri"
+)
+
 func NewRunCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "run",
 		Short: "Run the OpenFGA server",
 		Long:  "Run the OpenFGA server.",
 		Run:   run,
 		Args:  cobra.NoArgs,
+		PreRun: func(cmd *cobra.Command, args []string) {
+			flags := cmd.Flags()
+
+			util.MustBindPFlag("experimentals", flags.Lookup("experimentals"))
```

Comment:
Why did we move these in here instead of in `run/flags.go`? I think these are more aptly placed in their own file, because they litter this command setup code.

Reply:
`flags.go` has code that is supposed to be only for the `run` command, but it executes irrespective of what command is being used. This introduces inconsistent behavior if we decide to have other commands with the same flags.

See https://github.com/spf13/viper/issues/233#issuecomment-479336184

> Bind your viper flags in `PreRun() or Run()` -- **NOT** `init()`.

(In our case we don't have `init()`, but the binding still runs in every case)

Reply:
It can remain as a `PreRun` func, but the organization of the code in its own file creates better code structure and organization.

```go
// cmd/run/flags.go

func BindRunFlagsFunc(cmd *cobra.Command, args []string) {
    ...
    util.MustBindPFlag(...) // pflag bindings specific to `openfga run`.
}
```
and then in, for example, `cmd/run/run.go`
```go
cmd := &cobra.Command{
    ...
    PreRun: cmd.BindCmdFlagsFunc,
    ...
```

Reply:
If I apply your suggestion, the declaration of flags will be in `NewRunCommand()` but the binding will be in `BindRunFlagsFunc` which is in a different file. So if someone tries to add a new flag, the change will be spread across  two files instead of one. I don't se the benefit of that 🤔

Reply:
What's wrong with that? It's within the same package. Go source files organize code in a logical way within the same package. The alternative and anti-practice would be to put everything within a single Go file.

Reply:
> What's wrong with that? 

There's nothing wrong, i'm asking what is the benefit. That file will only be changed when we add a new flag.

Reply:
The benefit to me as a maintainer is that it's easier to know to go to `flags.go` to change flag registration than it is to go to the PreRun function body of the `run.go` file.

Reply:
Updated all commands so that each have their own `flags.go` file